experimental/packages/api-logs/src/types/Event.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface Event {
+  /**
+   * The time when the event occurred as UNIX Epoch time in nanoseconds.
+   */
+  timestamp?: number;
+
+  /**
+   * The name of the event.
+   */
+  name: string;
+
+  /**
+   * The domain the event belongs to.
+   */
+  domain?: string;
+
+  /**
+   * Additional attributes that describe the event.
+   */
+  attributes?: Attributes;
```

Comment:
We still need to resolve the `nested` attribute issue, but this doesn't need to block this initial PR.